app/src/main/res/layout/login_fragment.xml

```diff
@@ -50,6 +50,13 @@
 
                 </com.google.android.material.textfield.TextInputLayout>
 
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/show_hint"
+                    android:textColor="#068AC6"
```

Comment:
color should not be hardcoded like strings... use @color/

Reply:
Yes I did not want to make these changes for now I will be careful next time while committing.